src/jobs/aws_deploy.yml

```diff
@@ -9,20 +9,32 @@ parameters:
   aws_profile:
     type: string
     description: "The AWS profile to be used"
-  deployment_type:
-    type: string
-    description: "The AWS deployment type"
-  cluster_name:
-    type: string
-    description: "The Cluster name to be used"
+    default: "default"
   region:
     type: string
     description: "The aws region for the deployment"
+    default: eu-west-1
+  credentials_file_content:
+    type: string
+    default: |
+      [default]
+      aws_access_key_id=$ACCESS_KEY_ID
+      aws_secret_access_key=$SECRET_ACCESS_KEY
+      [integration]
+      source_profile = default
+      role_arn = arn:aws:iam::$AWS_ACCOUNT_ID_INTEGRATION:role/$CONTINUOUS_DELIVERY_ROLE
+      region=eu-west-1
```

Comment:
should this come from the parameters?

Reply:
I mean this should probably come from an env var right?

Reply:
is just that this way you can override it if you feel that you need a different set of credentials

Reply:
if you specify it in the parameters I do expect it to flow accordingly, I can do the change in case.

Reply:
solved